go/vt/topo/locks.go

```diff
@@ -127,21 +127,6 @@ var locksKey locksKeyType
 // - a context with a locksInfo structure for future reference.
 // - an unlock method
 // - an error if anything failed.
-//
-// We lock a keyspace for the following operations to be guaranteed
-// exclusive operation:
-// * changing a keyspace sharding info fields (is this one necessary?)
-// * changing a keyspace 'ServedFrom' field (is this one necessary?)
-// * resharding operations:
-//   - horizontal resharding: includes changing the shard's 'ServedType',
-//     as well as the associated horizontal resharding operations.
-//   - vertical resharding: includes changing the keyspace 'ServedFrom'
-//     field, as well as the associated vertical resharding operations.
-//   - 'vtctl SetShardIsPrimaryServing' emergency operations
-//   - 'vtctl SetShardTabletControl' emergency operations
-//   - 'vtctl SourceShardAdd' and 'vtctl SourceShardDelete' emergency operations
-//
-// * keyspace-wide schema changes
 func (ts *Server) LockKeyspace(ctx context.Context, keyspace, action string) (context.Context, func(*error), error) {
```

Comment:
Removing this outdated comment. We should be using the IDE to find all Usages rather than maintaining such a list here.